scripts/check-cxx-interop-compatibility.sh

```diff
@@ -37,6 +37,19 @@ for product in $library_products; do
   echo "import $product" >> "$source_file"
 done
 
+# A platform independent (but brittle) way to add the swift settings block
```

Comment:
Can't we use the fact that `Package` has reference semantics and just add settings after the fact?

i.e something like:

```swift
let package = Package(...)

// Generate this bit:
for target in package.targets {
  target.swiftSettings = [....]
}